app/views/trainees/show.html.erb

```diff
@@ -1,59 +1,86 @@
+        :row,
+        task_name: 'Record setup',
+        path: "",
+        status: 'completed'
+      )
+    end %>
 
-<h2 class="govuk-heading-l"><%= t('views.trainee_summary.contact_details.heading') %></h2>
+    <h2 class="govuk-heading-m">Trainee details</h2>
 
-<p class="govuk-body">
-  <%= govuk_link_to "Update", contact_details_trainee_path(@trainee) %>
-</p>
+    <%= render_component TaskList::View.new(classes: "personal-details") do |component|
+      component.slot(
+        :row,
+        task_name: 'Personal details',
+        path: personal_details_trainee_path(@trainee),
+        status: 'completed'
+      )
 
-<div class="contact-details">
-  <%= render_component(SummaryCard::View.new(title: "Contact details",
-                                             rows: TraineeContactDetails.new(@trainee).call)) %>
-</div>
+      component.slot(
+        :row,
+        task_name: 'Contact details',
+        path: contact_details_trainee_path(@trainee),
+        status: 'in progress'
+      )
 
-<h2 class="govuk-heading-l">Previous education</h2>
+      component.slot(
+        :row,
+        task_name: 'Diversity information',
+        path: '#details',
+        status: 'not started'
+      )
+    end %>
 
-<p class="govuk-body">
-  <%= govuk_link_to "Update", previous_education_trainee_path(@trainee) %>
-</p>
+    <h2 class="govuk-heading-m">Education</h2>
 
-<div class="previous-education">
-  <%= render_component(SummaryCard::View.new(title: "Previous education",
-                                             rows:  TraineePreviousEducation.new(@trainee).call)) %>
-</div>
+    <%= render_component TaskList::View.new(classes: "education") do |component|
+      component.slot(
+        :row,
+        task_name: 'GCSE',
+        path: "#",
+        status: 'completed'
+      )
 
-<h2 class="govuk-heading-l">Course details</h2>
+      component.slot(
+        :row,
+        task_name: 'Degree',
+        path: "#",
+        status: 'in progress'
+      )
+    end %>
 
-<p class="govuk-body">
-  <%= govuk_link_to "Update", course_details_trainee_path(@trainee) %>
-</p>
+    <h2 class="govuk-heading-m">Assessment</h2>
 
-<div class="course-details">
-  <%= render_component(SummaryCard::View.new(title: "Course details",
-                                             rows:  TraineeCourseDetails.new(@trainee).call)) %>
+    <%= render_component TaskList::View.new(classes: "assessment") do |component|
+      component.slot(
+        :row,
+        task_name: 'Assessment details',
+        path: course_details_trainee_path(@trainee),
+        status: 'completed'
+      )
+    end %>
+  </div>
 </div>
 
-<h2 class="govuk-heading-l">Training details</h2>
-
-<p class="govuk-body">
-  <%= govuk_link_to "Update", training_details_trainee_path(@trainee) %>
-</p>
+<%= form_with url: trn_submissions_path, method: :post, local: true do |f| %>
+  <%= hidden_field_tag :trainee_id, @trainee.id %>
+  <%= f.govuk_submit "Check and submit for TRN" %>
```

Comment:
This renders out as an input - should it be a button?

Reply:
good spot, more of a issue for `govuk_design_system_formbuilder`

Reply:
https://github.com/DFE-Digital/govuk_design_system_formbuilder/issues/203